app/models/region.rb

```diff
@@ -26,6 +26,18 @@ def self.root
     Region.find_by(region_type: :root)
   end
 
+  def slug_candidates
+    [
```

Comment:
Is this data-structure something that FriendlyID understands?

Reply:
According to [this doc](https://norman.github.io/friendly_id/file.Guide.html#Candidates) it would appear yes.